src/test/java/com/stripe/functional/StripeObjectTest.java

```diff
@@ -22,15 +22,6 @@ public void testGetRawJsonObject() throws StripeException {
         subscription.getCreated(),
         subscription.getRawJsonObject().getAsJsonPrimitive("created").getAsLong());
 
-    // Access `plan[id]`, a nested string element
```

Comment:
`plan` top-level has been undocumented on `Subscription` and there's no equivalent hash that is set. I'm assuming that since we look deeper in the hash below (for `items`) then we don't need this test. @ob-stripe are you okay with this test being removed?